examples/tensorflow2/scripts/tf2_save_metrics.py

```diff
@@ -0,0 +1,215 @@
+"""
+This file is temporary, for testing with 2.X.
```

Comment:
so these files will be deleted just before merge?

Reply:
These files are currently run on the AWS TF test pipeline.
They will be either modified or deleted after merging.